libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-kotlin = "1.6.10"
+kotlin = "1.6.10" # bumping to 1.6.20 breaks IDE autocomplete
```

Comment:
OPened https://youtrack.jetbrains.com/issue/KT-52064/1.6.20-break-IDE-autocomplete-in-Android-module-depending-on-mul about this